conda/recipes/librmm/recipe.yaml

```diff
@@ -110,3 +110,54 @@ outputs:
       homepage: ${{ load_from_file("python/librmm/pyproject.toml").project.urls.Homepage }}
       license: ${{ load_from_file("python/librmm/pyproject.toml").project.license.text | replace(" ", "-") }}
       summary: librmm test & benchmark executables
+  - package:
```

Comment:
@gforsyth I'd like your eyes on this if you don't mind. We build the examples separately from the rest of the C++ library so that we can test that the library builds actually work (and are findable) -- but this seems to require that we re-state the `env:` here. I tried using YAML anchors to re-use the `env:` key but rattler rejected that syntax. Is there a better way you can think of? Or should we just skip using sccache for building the examples?

Reply:
Was it failing without the re-stated `env` block?  That shouldn't be required -- it's supposed to be inherited

Reply:
Hmm. I think it failed to find CUDA compilers. I'll push that change here and see if it reproduces in CI.

Reply:
@gforsyth Here's the errors I see in CI: https://github.com/rapidsai/rmm/actions/runs/13509151290/job/37745645040?pr=1800#step:9:4904

Reply:
It also has more errors:

```
CMake Error: CMAKE_CXX_COMPILER not set, after EnableLanguage
CMake Error: CMAKE_CUDA_COMPILER not set, after EnableLanguage
```

Reply:
Ugh, I guess I'm wrong.

I'm annoyed that this is the behavior, given how the outputs all inherit runtime dependencies.  It built locally for me without the extra `env` block, but I'll wager that's only because I have `make` on my `$PATH`

Reply:
The example compilation is pretty light, so maybe we skip `sccache` here just to avoid the extra `env` block, but for other recipes we'll have to include it.

Reply:
I think I'm going to keep `sccache` just for consistency with the `env:` above. RMM is often a source that people copy from, and I'd rather let readers learn to use `sccache` here for the complex cases.

Reply:
Reverted in edbd54e9.